README.md

```diff
@@ -1,24 +1,59 @@
+## Setting up your Environment
+* In order to work on the **Operation Code** site, you will need to install a few things.
 
-* Ruby version
+  ### Docker
+  Docker is a container system. Click the link below to install Docker on your Operating System.
 
-* System dependencies
+  * [Mac](https://www.docker.com/docker-mac)
+  * [Windows](https://www.docker.com/docker-windows)
+  * [Linux (Ubuntu 16.04)](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04)
 
-* Configuration
+  ### Git
+  Git is a distributed version control system. This is how our code is stored and managed. Git can be frustrating, but it is an essential tool. If you want to learn more about Git, a great resource is [Think Like a Git](http://think-like-a-git.net/). If you find yourself in a real git pickle, see ["Oh, shit, git!"](http://ohshitgit.com/).
 
-* Database creation
+  * [Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 
-* Database initialization
+  ### Make (for Windows only)
+  Make is a tool which controls the generation of executables and other non-source files of a program from the program's source files.  It is a standard part of the GNU library and is included in most distrubtions of Linux, as well as OSX.  Unfortunately, Windows does not have a native make program, so it will need to be installed and added to PATH in order for the command line commands to work.  You can use the [GNUWin general installation guide](http://gnuwin32.sourceforge.net/install.html) as a general guide on how to install a GNU package and add it to PATH.
 
-* How to run the test suite
+  * [Install Make](http://gnuwin32.sourceforge.net/packages/make.htm)
 
-* Services (job queues, cache servers, search engines, etc.)
+  ### Operation Code
+  You are now ready for the actual **Operation Code** code base.
+
+  * The common practice is to make a copy of the GitHub repository you want to work on (known as `forking` the repo), make your changes, and then request to merge those changes back into the project (known as a `pull request`).
+  * Forking a repo is done through GitHub's web UI. It can be found in the top right corner of the **Operation Code**'s [GitHub](https://github.com/OperationCode/operationcode_backend) page.
+
+  * The following commands will pull down the source code from your forked repo.
+  * Make sure to replace `[YOUR-GITHUB-NAME]` with your GitHub name.  (example: https://github.com/iserbit/operationcode.git)
+
+  #### Local Development Environment
+    _Fork the repo first._
+    ```bash
+    git clone https://github.com/[YOUR-GITHUB-NAME]/operationcode_backend.git operationcodebackend-upstream
+    cd operationcodebackend-upstream
+    git remote add upstream https://github.com/OperationCode/operationcode_backend.git
+    ```
+
+  #### Database Setup:
+  To setup the database simply run the following command in the root directory of the project.
+
+  ```bash
+  make setup
```

Comment:
You shouldn't need postgres app. docker-compose is pulling in postgres as a container and wiring up networking so it works with rails.

It's certainly possible to get things running without docker-compose and it would be nice to document that process as 'advanced setup' but there's a few more steps that we need to take.